kolibri/plugins/management/assets/src/actions.js

```diff
@@ -320,19 +328,31 @@ function cancelImportExportWizard(store) {
 
 // called from a timer to continually update UI
 function pollTasksAndChannels(store) {
-  TaskResource.getCollection().fetch({}, true).then(
+  const samePageCheck = samePageCheckGenerator(store);
+  TaskResource.getCollection().fetch({}, true).only(
+    () => {
```

Comment:
Could be reduced to a return expression I think?

`() => samePageCheck() && !store.state.pageState.wizardState.busy`